Formula/perl.rb

```diff
@@ -55,6 +55,20 @@ def install
     system "make", "install"
   end
 
+  def post_install
+    on_linux do
+      perl_core = Pathname.new(`#{bin/"perl"} -MConfig -e 'print $Config{archlib}'`)+"CORE"
```

Comment:
Can we move the backticks to a separate line and capture it with `safe_popen_read` for use in this line? This one's a bit hard to read.

Reply:
Done. We have a similar safe_popen_read in the subversion formula, I copied it from there.